docker-compose.ci.yml

```diff
@@ -22,7 +22,7 @@ services:
     depends_on:
       - ch-server
   ch-server:
-    image: clickhouse/clickhouse-server:23.8.8-alpine
+    image: clickhouse/clickhouse-server:head-alpine
```

Comment:
?

Reply:
To be aligned with other compose files; actually we can probably remove this service for now. I will clean this up in the following PR